integration/simulation/utils.go

```diff
@@ -201,3 +204,7 @@ func findRollupDups(list []common.L2RootHash) map[common.L2RootHash]int {
 	}
 	return dups
 }
+
+func SleepRndBtw(min time.Duration, max time.Duration) {
```

Comment:
This ended up here since it was only used by the sims, so didn't need to go into the `integration/common/` package.